namui/src/getGcCanvasKitPackage.ts

```diff
@@ -45,3 +52,14 @@ function makeItGc<T extends EmbindObject<any>>(
 
   return gcConstructor as any;
 }
+
+function makeItGcAlt<T extends EmbindObject<any>, P extends any[]>(
```

Comment:
I believe this would be merge with above function(`makeItGc`) because params always passed in `makeItGc` too. Could you check it and update it?